cmd/common/containerized-gadgets/helpers.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2023 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package containerizedgadgets
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/viper"
+	"oras.land/oras-go/v2/content/oci"
+	"oras.land/oras-go/v2/registry/remote/auth"
+)
+
+func getLocalOciStore() (*oci.Store, error) {
+	home, err := os.UserHomeDir()
```

Comment:
What about using `/var/lib`?

Reply:
I would argue that it depends. The currently added commands don't require being root.

Do we want to enforce that?

Reply:
Let's keep using $HOME for now then. I think it's a parameter that we'll make configurable later on anyway.